### ISSUE TITLE
Enchancement Request: Streamlined features under a file

### ISSUE THREAD

**Issue by this-is-yaash** [AmateursLeague/sneaky-package#21]
So far, this package has 3 features:
1. `models.py` - Displays the source code/information of the files in the console output.
2. `graph.py` - Copies the source code/information to the system's clipboard.
3. `piechart.py` - Writes the file to the root directory of execution.

I want these features combined into a single file.

**Example:**
```python
from package_name.models import display
```
- `models` - the feature/file (`models.py`)
- `display` - the method inside the feature/file

I want all three features inside a single file. (I'll leave the naming conventions to you!)  
It should execute like this:

```python
from package_name.newFeatureName import feature1
feature1('filename')
```

In simple terms, instead of calling 3 features separately, we’ll call the methods separately from one file."

**Comment by KirtiPratihar** [comment]
@this-is-yaash I would love to work on this. can you please assign it to me 

Thanks

**Comment by this-is-yaash** [comment]
@KirtiPratihar, any updates on your progress?

**Comment by KirtiPratihar** [comment]
Yes, I have added a file snippet_util.py. Please check the pull requst
